starknet_py/tests/e2e/fixtures/accounts.py

```diff
@@ -34,12 +30,13 @@
 async def devnet_account_details(
     account: BaseAccount,
     class_hash: int,
-    network: str,
+    run_devnet,
 ) -> Tuple[str, str]:
     """
     Deploys an Account and adds fee tokens to its balance (only on devnet).
     """
 
+    network = run_devnet
```

Comment:
I'm not convinced of this solution, any better idea?

Reply:
I'm not sure I understand this change tbh

Reply:
I removed [fixture network]( https://github.com/software-mansion/starknet.py/pull/1328/files#diff-d260393b7dd1835a3d244b5250f275d5a1802a4695b30c3e195426a296ac4394L52) which was used in a few places. Currently, we need only the devnet URL which can be obtained by calling `run_devnet` but IMO it's a little bit confusing:
```
@pytest.fixture(name="client", scope="package")
def create_full_node_client(run_devnet) -> FullNodeClient:
    return FullNodeClient(node_url=run_devnet + "/rpc")
```

Reply:
what is `run_devnet`s type? is this a function or a string? can we just pass the url, if this is all we need?

Reply:
It's Generator[str, None, None], so if you call run_devnet you will get a string 

```
@pytest.fixture(scope="package")
def run_devnet() -> Generator[str, None, None]:
    """
    Runs devnet instance once per module and returns it's address.
    """
    devnet_port, proc = start_devnet()
    yield f"http://localhost:{devnet_port}"
    proc.kill()
```

Reply:
For me it can be either `run_devnet_and_get_url` or simply `devnet`. In the second option, I would pass `devnet` directly without the assignment `network = devnet`.

Reply:
let's do that then